parm/metplus/GridStat_APCPgt01h.conf

```diff
@@ -257,13 +257,13 @@ GRID_STAT_OUTPUT_FLAG_NBRCNT = STAT
 
 # NetCDF matched pairs output file
 #GRID_STAT_NC_PAIRS_VAR_NAME =
-GRID_STAT_NC_PAIRS_FLAG_LATLON = TRUE
-GRID_STAT_NC_PAIRS_FLAG_RAW = TRUE
+GRID_STAT_NC_PAIRS_FLAG_LATLON = FALSE
```

Comment:
Just curious why we're turning off `GRID_STAT_NC_PAIRS_FLAG_LATLON`, `GRID_STAT_NC_PAIRS_FLAG_RAW`, and `GRID_STAT_NC_PAIRS_FLAG_NBRHD` (see [here](https://met.readthedocs.io/en/latest/Users_Guide/grid-stat.html?highlight=nc_pairs_flag%20latlon)).